apps/openmw/mwclass/misc.cpp

```diff
@@ -85,7 +86,22 @@ namespace MWClass
         {
             const ESM::Creature *creature = MWBase::Environment::get().getWorld()->getStore().get<ESM::Creature>().search(ref->mRef.getSoul());
             if (creature)
-                value *= creature->mData.mSoul;
+            {
+                int soul = creature->mData.mSoul;
+                if (Settings::Manager::getBool("rebalance soul gem values", "Game"))
+                {
+                    // use the 'soul gem value rebalance' formula from the Morrowind Code Patch 
+                    float soulValue = 0.0001 * pow(soul, 3) + 2 * soul;
+                    
```

Comment:
Superfluous whitespace here.

Reply:
In C++, all whitespace is superfluous. ;)
One line doesn't hurt readability.